static_rdf_server/utils/utils.py

```diff
@@ -27,30 +36,28 @@ async def decide_content_and_extension(
     extension: str = "html"
 
     # We inspect the accept-header:
-    if accept_header:
-        if "text/turtle" in accept_header:
-            content_type = "text/turtle"
-            content_language = ""
-            extension = "ttl"
-        elif "text/html" in accept_header:
-            pass
-        elif "*/*" in accept_header:
-            pass
-        else:
+    if len(accept_header) > 0:
+        try:
+            content_type = decide_content_type(accept_header, supported_content_types)
+            if content_type == "text/turtle":
+                extension = "ttl"
+            elif content_type == "text/html":
+                extension = "html"
```

Comment:
Ikke kjempehappy med hardkoda verdier her, som _må_ matche med DEFAULT_blablabla ovenfor. Kan vi unngå det på noe vis?

Reply:
Kan lage et map som ligger sammen med defaults, og så benytte map her for å finne extension. Da er det ganske åpenbart at man skal endre defaults og map samtidig, og alt er på ett sted.

Reply:
Fiksa i 12608887e214f132a073f0be5d944972101cf07d